api/c/indigo/src/indigo_internal.h

```diff
@@ -200,8 +200,7 @@ class DLLEXPORT IndigoObject
 
 protected:
     AutoPtr<Array<char>> _dbg_info; // allocated by debugInfo() on demand
-private:
-    IndigoObject(const IndigoObject&);
+    IndigoObject(const IndigoObject&){};
```

Comment:
Doubtful change. If we want to have copy-constructor, it's better to leave it for compiler by default. If we want to override the default constructor, at least `_type` should be copied.

Reply:
Missed the 'type' field. Fixed.